cmake/HPX_SetupStdexec.cmake

```diff
@@ -0,0 +1,34 @@
+if(STDEXEC_ROOT AND NOT Stdexec_ROOT)
+    set(Stdexec_ROOT
+        ${STDEXEC_ROOT}
+        CACHE PATH "STDEXEC base directory"
+    )
+    unset(STDEXEC_ROOT CACHE)
+endif()
```

Comment:
We have this kind of code in our CMake build system only to be backwards compatible, not sure if we need this at all.

Reply:
Makes sense, removing